service/src/main/java/bio/terra/workspace/app/controller/ReferencedGcpResourceController.java

```diff
@@ -661,6 +661,7 @@ public ResponseEntity<ApiCloneReferencedGcpGcsObjectResourceResult> cloneGcpGcsO
                 sourceReferencedResource,
                 body.getDestinationWorkspaceId(),
                 UUID.randomUUID(), // resourceId is not pre-allocated for individual clone endpoints
+                /*destinationFolderId=*/ null,
```

Comment:
Same for rest of this file.
```
// Pass in null for now, since we don't know destination folder id yet. Folder id will be set in a step.
/*destinationFolderId=*/ null,
```